dashboardsSanitizer/build.gradle

```
@@ -0,0 +1,38 @@
+buildscript {
+    dependencies {
+        classpath 'org.junit.platform:junit-platform-gradle-plugin:1.0.1'
+    }
+}
+
+plugins {
+    id 'application'
+    id 'org.opensearch.migrations.java-library-conventions'
+    id 'io.freefair.lombok'
+    id 'java'
+}
+
+java.sourceCompatibility = JavaVersion.VERSION_11
+java.targetCompatibility = JavaVersion.VERSION_11
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+
+    implementation project(":commonDependencyVersionConstraints")
+
+    implementation project(':coreUtilities')
+    implementation project(":RFS")
+
+    implementation group: 'com.fasterxml.jackson.core', name:'jackson-databind'
+    implementation group: 'org.slf4j', name: 'slf4j-api'
+    implementation group: 'org.apache.logging.log4j', name: 'log4j-slf4j2-impl'
+
+    implementation group: 'info.picocli', name: 'picocli', version:'4.7.6'
```

Please replace with the existing CLI library we use in this repo.
```suggestion
    implementation group: 'com.beust', name: 'jcommander'
```